src/components/va-alert/va-alert.tsx

```diff
@@ -128,33 +128,37 @@ export class VaAlert {
 
   render() {
     const { backgroundOnly, status, fullWidth, visible, closeable } = this;
-    const classes = `alert ${status} ${fullWidth ? 'full-width' : ''} ${
-      backgroundOnly ? 'bg-only' : ''
-    }`;
+    const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
     const role = status === 'error' ? 'alert' : null;
     const ariaLive = status === 'error' ? 'assertive' : null;
 
     if (!visible) return <div aria-live="polite" />;
 
     return (
       <Host>
-        <div role={role} aria-live={ariaLive} class={classes}>
-          <i aria-hidden="true" role="img"></i>
-          <div class="body" onClick={this.handleAlertBodyClick.bind(this)}>
-            {!backgroundOnly && <slot name="headline"></slot>}
-            <slot></slot>
+        <div class={fullWidth ? 'full-width' : ''}>
```

Comment:
The simplest way to limit the body width while letting the existing styles apply to the rest of the markup was to create a wrapping div. This div will be at full width, which means the top border is also full width. But this also allows the CSS to limit the width of the inner div.

Reply:
Did you choose to not use the `host:` selector because the changes it would require in the CSS would be pretty extensive?

Reply:
Originally I was just trying to do this quickly, thinking that I could get the fix out before the daily deploy. Now that I have more time I'll actually give that a shot :sweat_smile:

Reply:
I've changed it now so that it's based around `:host`.